umbraco/Umbraco-CMS
# Fix to ensure we use virtual paths for any blocks with custom views

Fixes AZ#7611
Will use a virtual path in case Umbraco put in an unusual place

## Test Notes
* Add a block to block list property editor that uses a custom HTML view
* Ensure it renders
* Add a block without a custom HTML view - ensure that renders the default one
